docs/next/guides/getting-started/setting-options.md

```diff
@@ -14,319 +14,219 @@ tags:
 
 ## Overview
 
-## Available options
-
-Opisać core i plugins i wrzucić linki do reference do obu
-
-Najlepiej jakby to było dostępne też dla wrapperów
-
-## The cascading configuration
+Handsontable uses cascading configuration, a fast way to provide configuration options for the entire table, including its columns and particular cells. To show you how it works, we use the [readOnly](@/api/metaSchema.md#readOnly) and [className](@/api/metaSchema.md#className) options, which we will play in all the examples below.
```

Comment:
which we will play in all the examples below. >>> whenever possible avoid future tenses in documentation (will etc.)

which we will play in all the examples below. > remove.